src/components/tutor-schedule/TutorSchedule.tsx

```diff
@@ -33,19 +34,25 @@ const items: ITutorScheduleItem[] = [
 ]
 
 function TutorSchedule() {
+  const { t } = useTranslation()
+
   return (
     <Box sx={styles.generalContainer}>
       <Box sx={styles.textContainer}>
-        <Typography sx={styles.sectionTitle}>Your schedule</Typography>
-        <Typography sx={styles.sectionSubtitle}>Upcoming classes</Typography>
+        <Typography sx={styles.sectionTitle}>
+          {t('tutorHomePage.schedule.title')}
+        </Typography>
+        <Typography sx={styles.sectionSubtitle}>
+          {t('tutorHomePage.schedule.subTitle')}
+        </Typography>
       </Box>
       <Box sx={styles.cardWrapper}>
         {items.map((item) => (
           <TutorScheduleCard item={item} key={item.lastName} />
         ))}
       </Box>
       <Button sx={styles.btn} variant='text'>
-        All scheduled classes
+        {t('tutorHomePage.schedule.AllClasses')}
```

Comment:
I think it's better to use lower camel case here.
E.g. `allClasses` instead of `AllClasses`.